lang/main.json

```diff
@@ -207,7 +207,9 @@
     "connectionindicator":
     {
         "header": "Connection data",
+        "connected_to": "Connected to:",
         "bitrate": "Bitrate:",
+        "bridge_count": "Server count: ",
```

Comment:
So I think maybe not using snake_case is okay. At some point the other snake_case strings should probably be changed to camelCase.

Reply:
oh, ok, I didn't know that.